### DISCUSSION TITLE
Feature Request: Day/Night Mode Switcher for Dynamic Wallpapers - Enhancing User Experience

### DISCUSSION THREAD

**Discussion by rdev32** [t1m0thyj/WinDynamicDesktop#561] [Ideas]
This app brought us dynamic wallpapers to change between the day.. but the last and more important thing is missing! The dark/light mode switcher. We should be able to program a switcher between day and night.
Please make it possible!

**Comment by t1m0thyj** [comment]
WinDD should already be able to switch the system theme between dark/light when PowerShell scripts are installed: https://github.com/t1m0thyj/WDD-scripts/tree/master/stable

Unfortunately these scripts don't seem to work very well in newer versions of Windows. As an alternative I recommend using another app [AutoDarkMode](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode) to achieve this functionality. It's lightweight and should coexist nicely with WinDD.
